internal/upload/awsupload/awsupload.go

```diff
@@ -74,7 +74,7 @@ func WaitUntilImportSnapshotTaskCompleted(c *ec2.EC2, input *ec2.DescribeImportS
 func WaitUntilImportSnapshotTaskCompletedWithContext(c *ec2.EC2, ctx aws.Context, input *ec2.DescribeImportSnapshotTasksInput, opts ...request.WaiterOption) error {
 	w := request.Waiter{
 		Name:        "WaitUntilImportSnapshotTaskCompleted",
-		MaxAttempts: 80, // this can take up to 15 minutes, therefore set the timeout to 20 minutes
+		MaxAttempts: 300, // this can take up to 15 minutes, therefore set the timeout to 20 minutes
```

Comment:
I'm not quite sure what this struct does but can we check for the status of the import instead of waiting an arbitrary amount of time and hoping everything works fine. Here's how it's done previously:
https://github.com/weldr/lorax/blob/master/tests/cli/test_build_and_deploy_aws.sh#L121

Reply:
`WaitUntilImportSnapshotTaskCompleted` is basically the loop you suggested with a timeout. I think there's no better solution. :-/